tools/ecfr-parser/eregs/supplemental_content.go

```diff
@@ -0,0 +1,38 @@
+package eregs
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/cmsgov/cmcs-eregulations/ecfr-parser/ecfr"
+)
+
+var SuppContentURL string
+
+func PostSupplementalPart(ctx context.Context, p ecfr.Part) (*http.Response, error) {
+	buff := bytes.NewBuffer([]byte{})
+	enc := json.NewEncoder(buff)
+	enc.SetEscapeHTML(false)
+
+	if err := enc.Encode(p); err != nil {
+		return nil, err
+	}
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, SuppContentURL, buff)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/json")
+	resp, err := client.Do(req)
+	if err != nil {
+		return resp, err
+	}
+
```

Comment:
To avoid double-closing consider returning only `error` from this function instead of doing error-handling in `handlePart`. That's what I changed in `eregs.PostPart` to avoid the same thing.